arcgis-ios-sdk-samples/Analysis/Viewshed (location)/ViewshedLocation.storyboard

```diff
@@ -1,412 +1,424 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
The switch and color picker cells do not have any constraints positioning their subviews. This results in the subviews not being placed properly:

![screen shot 2018-10-17 at 11 25 41 am](https://user-images.githubusercontent.com/2257493/47107946-7daaa080-d1ff-11e8-9334-4dd633e6958e.png)

Please either add constraints or use a non-custom table view cell style.

Reply:
Thanks for finding that!